packages/cozy-harvest-lib/src/components/TriggerManager.jsx

```diff
@@ -144,65 +113,14 @@ export class TriggerManager extends Component {
   async handleOAuthAccountId(accountId) {
     const { findAccount } = this.props
     try {
+      this.setState({ error: null, status: RUNNING })
```

Comment:
are we sure to want to unset the error state? Can't we have something inside?

Reply:
At this point we received an account id from stack and we are going to run, or re-run the konnector. We do not want to show an error anymore in the TriggerManager, if any.

Reply:
IF we had something in this error state, it would be from the previous attempt to connect the account. Since a new attempt just started, yes, we really want to unset it :)

Reply:
Thanks